personhood-oracle/src/lib.rs

```diff
@@ -117,10 +117,12 @@ decl_module! {
 
 decl_error! {
     pub enum Error for Module<T: Config> {
-        /// Only other parachains can call this function
-        OnlyParachainsAllowed,
```

Comment:
@clangenb why is this one not used (anymore?)

Reply:
When I refactored, I probably forgot to put it here: https://github.com/encointer/pallets/blob/ed78e1ad5a8f8995761c897fca9d4e0af9c328fd/personhood-oracle/src/lib.rs#L96

@NicJak please re-insert the `OnlyParachainsAllowed` in above line.

Reply:
Replaced UnableToDecodeRequest with OnlyParachainsAllowed